components/molecules/EntryBanner.js

```diff
@@ -0,0 +1,27 @@
+import Banner from "../atoms/Banner";
+import { ActionButton } from "../atoms/ActionButton";
+
+export const EntryBanner = () => {
+  return (
+    <section>
+      <Banner
+        siteTitle="Service Canada Labs"
+        headline="Make government work for you."
+      />
+      <div className="bg-gray-light-200 py-6">
+        <div className={"flex container xxs:mx-auto lg:px-6 xxl:mx-auto"}>
+          <ActionButton
```

Comment:
Again when these buttons are clicked... how are we supposed to respond to them if there is no callback ?

Reply:
These buttons are basically just links. They don't actually submit any data.

We should really use links here, although we can style them as buttons, or we can talk to Fred about changing how they look (better outcome).

Reply:
Actually, maybe we can leave them as buttons for now and just keep a backlog story about links that are buttons.